tests/translator/output/error_embedded_connectors.json

```
@@ -11,6 +11,12 @@
     "Destination.Arn is missing. ",
     "Resource with id [EventsRuleMissingSqsQueueUrl] is invalid. ",
     "Unable to create connector from AWS::Events::Rule to AWS::SQS:Queue; it's not supported or the template is invalid. ",
+    "Resource with id [MyFunction] is invalid. ",
+    "Attribute 'NoProperties.Properties' should be a map. ",
```

Missing "Connectors"

```suggestion
    "Attribute 'Connectors.NoProperties.Properties' should be a map. ",
```